tf_files/aws/data.tf

```diff
@@ -1,2 +1,290 @@
+    }
+
+    statement {
+      effect = "Allow"
+      actions = [
+          "ecr:*",
+          "cloudtrail:LookupEvents"
+      ],
+      resources = [ "*" ]
+    }
+
+    statement {
+      actions = [ "ec2:*" ]
+      effect = "Allow"
+      resources = [ "*" ]
+    }
+
+    statement {
+      effect = "Allow"
+      actions = [ "elasticloadbalancing:*" ]
+      resources = [ "*" ]
+    }
+
+    statement {
+      effect = "Allow"
+      actions = [ "cloudwatch:*" ]
+      resources = [ "*" ]
+    }
+
+    statement {
+      effect = "Allow"
+      actions = [ "autoscaling:*" ]
+      resources = [ "*" ]
+    }
+
+    statement {
+        effect = "Allow"
+        actions = [ "iam:CreateServiceLinkedRole" ]
+        resources = [ "arn:aws:iam::*:role/aws-service-role/spot.amazonaws.com/AWSServiceRoleForEC2Spot*" ]
+        condition = {
+          test = "StringLike"
+          variable = "iam:AWSServiceName"
+          values = [ "spot.amazonaws.com" ]
+        }
+    }
+
+    statement {
+        effect = "Allow"
+        actions = [ "iam:CreateServiceLinkedRole" ]
+        resources = [ "arn:aws:iam::*:role/aws-service-role/spotfleet.amazonaws.com/AWSServiceRoleForEC2Spot*" ]
+        condition = {
+          test = "StringLike"
+          variable = "iam:AWSServiceName"
+          values = [ "spotfleet.amazonaws.com" ]
+        }
+    }
+    
+    statement {
+        effect = "Allow"
+        actions = [ "iam:CreateServiceLinkedRole" ]
+        resources = [ "arn:aws:iam::*:role/aws-service-role/ec2scheduled.amazonaws.com/AWSServiceRoleForEC2Scheduled*" ]
+        condition = {
+          test = "StringLike"
+          variable = "iam:AWSServiceName"
+          values = [ "ec2scheduled.amazonaws.com" ]
+        }
+    }
+    
+    statement {
+        actions = [
+            "rds:*",
+            "cloudwatch:DescribeAlarms",
+            "cloudwatch:GetMetricStatistics",
+            "ec2:DescribeAccountAttributes",
+            "ec2:DescribeAvailabilityZones",
+            "ec2:DescribeInternetGateways",
+            "ec2:DescribeSecurityGroups",
+            "ec2:DescribeSubnets",
+            "ec2:DescribeVpcAttribute",
+            "ec2:DescribeVpcs",
+            "sns:ListSubscriptions",
+            "sns:ListTopics",
+            "sns:Publish",
+            "logs:DescribeLogStreams",
+            "logs:GetLogEvents",
+        ]
+        effect = "Allow"
+        resources = [ "*" ]
+    }
+
+    statement {
+        actions = [ "pi:*" ]
+        effect = "Allow"
+        resources = [ "arn:aws:pi:*:*:metrics/rds/*" ]
+    }
+
+    statement {
+        actions = [ "iam:CreateServiceLinkedRole" ]
+        effect = "Allow"
+        resources = [ "arn:aws:iam::*:role/aws-service-role/rds.amazonaws.com/AWSServiceRoleForRDS" ]
+        condition = {
+          test = "StringLike"
+          variable = "iam:AWSServiceName"
+          values = [ "rds.amazonaws.com" ]
+        }
+    }
+
+    statement {
+      effect = "Allow"
```

Comment:
Can we get rid of this? Or at least reduce the scope of it? This will allow you to do everything...?